src/vcl/Image.cc

```diff
@@ -31,19 +31,261 @@
 
 #include "vcl/Image.h"
 #include "vcl/Exception.h"
-#include "ImageData.h"
 
 using namespace VCL;
 
+    /*  *********************** */
+    /*        OPERATION         */
+    /*  *********************** */
+
+    /*  *********************** */
+    /*       READ OPERATION     */
+    /*  *********************** */
+
+Image::Read::Read(const std::string& filename, Image::Format format)
+    : Operation(format),
+      _fullpath(filename)
+{
+}
+
+void Image::Read::operator()(Image *img)
```

Comment:
See my comment about separating operators in a different file.

Reply:
I suggest creating a separate patch for it, as this set got already big enough.